reads/reads-data/create_short_tandem_repeat_reads_db.py

```diff
@@ -1,99 +1,86 @@
             `n_alleles` integer,
-            `allele_1_repeat_unit` text,
-            `allele_2_repeat_unit` text,
+            `allele_1_repeat_unit` varchar(24),
+            `allele_2_repeat_unit` varchar(24),
             `allele_1_repeats` integer,
             `allele_1_repeats_ci_lower` integer,
             `allele_1_repeats_ci_upper` integer,
             `allele_2_repeats` integer,
             `allele_2_repeats_ci_lower` integer,
             `allele_2_repeats_ci_upper` integer,
-            `population` text,
-            `sex` text,
-            `age` text,
-            `pcr_protocol` text,
-            `filename` text
+            `population` char(3),
+            `sex` char(2),
+            `age` varchar(17),
+            `pcr_protocol` char(8),
+            `filename` char(47),
+            `q` double,
+            `quality_description` varchar(12)
         )
         """
     )
 
     db.execute("CREATE INDEX `id_idx` ON `reads` (`id`)")
-
-    def _format_read(read, index, locus):
-        # Hemizygotes have only one allele and only one value in Genotype/GenotypeConfidenceInterval
-        if "/" in read["Genotype"]:
-            n_alleles = 2
-            allele_1_repeats, allele_2_repeats = map(int, read["Genotype"].split("/"))
-            allele_1_ci, allele_2_ci = read["GenotypeConfidenceInterval"].split("/")
-            allele_1_ci_lower, allele_1_ci_upper = map(int, allele_1_ci.split("-"))
-            allele_2_ci_lower, allele_2_ci_upper = map(int, allele_2_ci.split("-"))
-        else:
-            n_alleles = 1
-            allele_1_repeats = int(read["Genotype"])
-            allele_2_repeats = None
-            allele_1_ci_lower, allele_1_ci_upper = map(int, read["GenotypeConfidenceInterval"].split("-"))
-            allele_2_ci_lower = allele_2_ci_upper = None
-
-        if "/" in read["Motif"]:
-            allele_1_repeat_unit, allele_2_repeat_unit = read["Motif"].split("/")
-        else:
-            allele_1_repeat_unit = read["Motif"]
-            allele_2_repeat_unit = read["Motif"]
-
-        return {
-            "id": locus,
-            "order": index,
-            "n_alleles": n_alleles,
-            "allele_1_repeat_unit": allele_1_repeat_unit,
-            "allele_2_repeat_unit": allele_2_repeat_unit,
-            "allele_1_repeats": allele_1_repeats,
-            "allele_1_repeats_ci_lower": allele_1_ci_lower,
-            "allele_1_repeats_ci_upper": allele_1_ci_upper,
-            "allele_2_repeats": allele_2_repeats,
-            "allele_2_repeats_ci_lower": allele_2_ci_lower,
-            "allele_2_repeats_ci_upper": allele_2_ci_upper,
-            "population": read["Population"],
-            "sex": read["Sex"],
-            "age": None if read["Age"] == "age_not_available" else read["Age"],
-            "pcr_protocol": read["PcrProtocol"],
-            "filename": read["ReadvizFilename"] or None,
-        }
-
     for locus, reads in reads_data.items():
+        print(locus)
```

Comment:
Is this print statement leftover from debugging, or is this meant to be here? -- just wanted to do a quick double check.